### DISCUSSION TITLE
RayHit

### DISCUSSION THREAD

**Discussion by MrScautHD** [amerkoleci/JoltPhysicsSharp#13] [General]
Hi, i would ask how we can impliment RayHit. Oh and do you guys have a discord?

**Comment by amerkoleci** [comment]
It's a single development project, no discord atm sorry :) RayHit is missing, I'll add it once I have chance.

Cheers!

**Reply by MrScautHD** [reply]
nice thx!

**Reply by amerkoleci** [reply]
Version 1.6.3 has support for NarrowPhaseQuery.CastRay.

**Reply by MrScautHD** [reply]
Nice work!

**Reply by MrScautHD** [reply]
It does crash by using it...
![grafik](https://github.com/amerkoleci/JoltPhysicsSharp/assets/65916181/d02cb87f-10c9-49b4-9e2a-3967ec4f70e5)
